src/js/components/DateInput/DateInput.js

```diff
@@ -40,11 +43,14 @@ const DateInput = forwardRef(
       onChange,
       onFocus,
       value: valueArg,
+      messages,
       ...rest
     },
     refArg,
   ) => {
     const theme = useContext(ThemeContext) || defaultProps.theme;
+    const announce = useContext(AnnounceContext);
+    const { format: formatMessage } = useContext(MessageContext);
```

Comment:
To stay constant can we do `const { format } = useContext(MessageContext);`

Reply:
@britt6612 There's an existing `format` prop on this component

Reply:
@jeferson-sb my mistake just saw that thank you for clarifying! 👍

Reply:
did you run `yarn test -update`?